azure-pipelines-richnav.yml

```diff
@@ -46,20 +46,30 @@ jobs:
   timeoutInMinutes: ${{ parameters.timeout }}
   variables:
     EnableRichCodeNavigation: true
+    Configuration: Debug
 
   steps:
     - task: PowerShell@2
       displayName: Build
       inputs:
         filePath: eng/build.ps1
-        arguments: -ci -restore -build -configuration Debug -prepareMachine
+        arguments: -ci -restore -build -configuration $(Configuration) -prepareMachine -pack /p:DotNetUseShippingVersions=true
```

Comment:
is this something we need to monitor on rolling builds for the infra side for failures?  Or, if it isn't an extremely long pipeline, could we run it on PRs?

Reply:
Right now it's quite long so it's not something we want to run in PRs. And yes, if this fails we should have some monitoring of it, but the first line would be more to email the owners of the LSIF tool and the indexing service than the swat team.

Reply:
Most of t he runs seem to be ~20 minutes which doesn't seem too long - 
![image](https://github.com/dotnet/roslyn/assets/5749229/99bb37a1-b14f-4d15-b28c-d95207cb5df6)

> but the first line would be more to email the owners of the LSIF tool and the indexing service than the swat team.

I was thinking if we're using the bootstrap version and introduced some regression

Reply:
@dibarbet That's because right now it's broken. :smile: It also turns out the queue reports green even if the LSIF generator crashes, which is something they're also going to have to fix for us.